transport/shadowsocks/cipher.go

```diff
@@ -26,8 +26,7 @@ import (
 	"golang.org/x/crypto/hkdf"
 )
 
-type Cipher struct {
```

Comment:
Maybe I was wrong, but I remembered `Cipher` is already referenced in tun2socks right?

Reply:
Yes, this is a breaking change. tun2socks is referring to the current version, so it doesn't break. I need to submit this PR first before I can update the dependency there.

Reply:
Update at https://github.com/Jigsaw-Code/outline-ss-server/pull/152